example/broker.conf

```diff
@@ -1,7 +1,7 @@
 # Default configuration for MQTT-CPP Broker
 verbose=1
-certificate=broker.crt.pem
-private_key=broker.key.pem
+certificate=server.crt.pem
+private_key=server.key.pem
```

Comment:
The following comment should be added. Please update to better English :)

# Automatic reload certificate and private_key.
# It is useful for Let's Encrypt certs update.
# 3600 means 3600seconds. If not set or set to 0, then no reload.
# certificate_reload_interval=3600